tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -1018,8 +1019,12 @@ def test_cluster_list_all_pythonic(self, cluster):
             original_username=original_username,
         ):
             # create dummy terminated cluster - set the daemon status to terminated, which will also
-            # update the cluster status
-            terminated_cluster = rh.cluster(name="terminated-cluster", ips=None).save()
+            # update the cluster status. Making sure that its name is in the same format as all testing clusters
+            # (contains timestamp and uuid)
+            terminated_cluster = rh.cluster(
```

Comment:
any reason we're adding `server_connection_type` and removing `ips` here?

Reply:
Yeah, so because the `ips` were set to `None`, the `alt_options` we set in the cluster factory method are an empty dictionary. Since we are deleting the resources every 24 hours, we are getting an exception when we try to load the terminated cluster from name => we are catching it and raising it again since there are no `alt_options`. 

Since it is a dummy cluster created just for the purpose of testing, I added `server_connection_type` so the `alt_options` will not be an empty dict. We don't really need to set the values of the ips to be explicitly set to `None` since it is the default behaviour. Therefore we are treating the cluster as terminated, which serves the purpose of this test.

[Code reference
](https://github.com/run-house/runhouse/blob/7caf3898e91c6e01bbb22a824019943fad3314b7/runhouse/resources/hardware/cluster_factory.py#L103)